test/test.js

```diff
@@ -25,6 +30,18 @@ describe('Walder', function () {
       expect(() => new Walder()).to.throw('Configuration file is required.')
     });
 
+    it('should be rejected with an error when the config file contains missing HTML files in a route', function () {
+      const configFile = path.resolve(__dirname, CONFIG_FILE_MISSING_HTML);
+      const walder = new Walder(configFile);
+      return walder.activate().should.be.rejectedWith(Error);
```

Comment:
Is it not possible to do this without needing `chai-as-promised`? See https://mochajs.org/#using-async-await

Reply:
Yup, did that.